requirements/local.txt

```diff
@@ -28,8 +32,10 @@ pytest==3.1.3             # via pytest-flask
 python-dateutil==2.6.1    # via alembic
 python-editor==1.0.3      # via alembic
 python-terraform==0.8.6
```

Comment:
Still needed?

Reply:
See above :) This is generated from `base.in`.